python/ray/experimental/dag/class_node.py

```diff
@@ -1,26 +1,51 @@
 import ray
-from ray.experimental.dag.dag_node import DAGNode
+import ray.experimental.dag as ray_dag
```

Comment:
```suggestion
from ray.experimental.dag import DAGNode, INPUT
```
better to do specific imports

Reply:
that's what i had but it will lead to circular imports warning so i just left public imports at dag __init__ level and just import the module here